CVs/input4MIPs_source_id.json

```diff
@@ -353,6 +353,27 @@
         "mip_era":"CMIP7",
         "source_version":"2.0"
     },
+    "FZJ-1-0":{
+        "activity_id": "input4MIPs",
+        "authors": [
+            {
+                "name": "Michaela Hegglin",
+                "email": "m.i.hegglin@fz-juelich.de",
+                "affiliations": [
+                    "Institute of Energy and Climate Research – Stratosphere (IEK-7), Forschungszentrum Jülich GmbH, 52425 Jülich, Germany",
```

Comment:
I also wonder whether the umlauts are going to cause a problem with HTML parsing, my guess is a yes